src/GslCore/AstTypes.fs

```diff
@@ -215,48 +228,54 @@ and AstNode =
     // to explode them into their outer contexts.
     | Splice of AstNode []
     with
-    /// Get the position from a node.
-    member x.pos =
+    /// Add a new position to this node.  This occurs during function expansion when an instance

```

Comment:
This docstring appears to refer to a different (possibly now deleted?) method?

Reply:
Yep - removed